src/lib.rs

```diff
@@ -362,9 +362,31 @@ fn builder_for_struct(ast: syn::MacroInput) -> quote::Tokens {
         )
     };
 
+    let builder_struct_doc = format!(
+        "Builder for {s}.\n\
+        \n\
+        # Examples\n\
+        \n\
+        ```\n\
```

Comment:
> Or do you generally prefer explicitness here?

Yes.

> Surprisingly I can't find this doc test in the output of cargo test.

I assume you made an example crate for this? Can you find the docs in `cargo doc`?